container/docker/handler.go

```diff
@@ -252,9 +252,11 @@ func newDockerContainerHandler(
 	// split env vars to get metadata map.
 	for _, exposedEnv := range metadataEnvs {
 		for _, envVar := range ctnr.Config.Env {
-			splits := strings.SplitN(envVar, "=", 2)
-			if splits[0] == exposedEnv {
-				handler.envs[strings.ToLower(exposedEnv)] = splits[1]
+			if envVar != "" {
+				splits := strings.SplitN(envVar, "=", 2)
+				if splits[0] == exposedEnv {
```

Comment:
Touché! If there's no equal sign, ```splits``` will be a single element list so the comparison with splits[1] would cause the application to fail. An ```len(splits) == 2``` is the only reasonable case, as my previous assumption that ```len(splits)``` could ever be more than 2 is wrong considering that the split happens only on the first equals. I'll update accordingly.